python/tvm/relay/frontend/pytorch.py

```diff
@@ -1016,17 +978,58 @@ def from_pytorch(script_module, input_shapes):
         TorchScripted PyTorch graph
         Note: We currently only support traces (ie: torch.jit.trace(model, input))
 
-    shape : Dictionary of input dimensions
+    input_shape : Dictionary of input dimensions
         Graph level input shape dictionary
+        The keys should be the same one returned by get_graph_input_names(...) above
 
     Returns
     -------
     mod : tvm.relay.Module
         The module that optimizations will be performed on.
 
-    params : dict of str to tvm.runtime
-        Dict of converted parameters stored in tvm.runtime format
+    params : dict of str to tvm.runtime.NDArray
+        Dict of converted parameters stored in tvm.runtime.ndarray format
     """
-    g = Graph(script_module, input_shapes)
-    mod, params = g.from_pytorch()
-    return mod, params
+    graph = script_module.graph.copy()
```

Comment:
Why a copy?

Reply:
Because we overwrite the input graph in place. This is not new in this PR.
Currently we apply the torch inlineing pass in `run_jit_passes(...)` below. I also do some surgery on the input graph in my QNN implementation.